actionpack/lib/action_controller/railtie.rb

```diff
@@ -31,7 +31,7 @@ class Railtie < Rails::Railtie #:nodoc:
             app.config.action_controller.delete(:always_permitted_parameters)
         end
         ActionController::Parameters.action_on_unpermitted_parameters = options.delete(:action_on_unpermitted_parameters) do
-          (Rails.env.test? || Rails.env.development?) ? :log : false
+          (Rails.env.test? || Rails.env.development?) ? :raise : false
```

Comment:
We can't change the default here. We need to take in consideration existing application that are going to upgrade. We need to add this new default to the `new_gramework_default` file, we need to make it the default when `load_defaults '6.0'` is called, but keep the `:log` default of the older versions.

Reply:
Yeah, I think it makes total sense.